### PR TITLE
Add example for service monitor in Helm chart

### PR DESCRIPTION
**Pull request by zhifanggao**
### Why are the changes needed?

Adding  an example to `servicemonitor` in values.yaml to improve the usability . 

### How was this patch tested?

execute command `helm install kyuubi-test .`
check the servicemonitor after deployment 

### Was this patch authored or co-authored using generative AI tooling?

No

### CODE REVIEW

File: charts/kyuubi/values.yaml [hunk 1]
```diff
@@ -289,6 +289,8 @@ metrics:
     enabled: false
     # List of service endpoints serving metrics to be scraped by Prometheus, see Prometheus Operator docs for more details
     endpoints: []
+    # Example for endpoints of services to be monitored
+    # - port: prometheus
```
[the remark this answers is not in the document]
Comment (zhifanggao) [reply charts/kyuubi/values.yaml]: fixed it

File: charts/kyuubi/values.yaml [hunk 2]
```diff
@@ -289,6 +289,8 @@ metrics:
     enabled: false
     # List of service endpoints serving metrics to be scraped by Prometheus, see Prometheus Operator docs for more details
     endpoints: []
+    #  endpoints:
+    #  - port: prometheus
```
Comment (pan3793) [inline charts/kyuubi/values.yaml:293]:
```suggestion
    #    - port: prometheus
```

**Review by pan3793 — APPROVED** [review]

**Comment by pan3793** [comment]
@zhifanggao please answer the last question in the PR template, this is mandated by the ASF and involves some legal issues. I will merge it after you complete that.

**Comment by zhifanggao** [comment]
> @zhifanggao please answer the last question in the PR template, this is mandated by the ASF and involves some legal issues. I will merge it after you complete that.

Done

**Comment by pan3793** [comment]
Thanks, merging to master

**Pull request closed without merging**